tests/UsersTest.php

```diff
@@ -228,6 +229,39 @@ public function testRemoveUserFromEverywhere()
         $this->assertSame('DELETED', $deletedUser['name'], 'User name has not been deleted');
     }
 
+    public function testsRemoveUserFromEverywhereFailsWhenLastUserInOrg()
+    {
+        $organization = $this->client->createOrganization($this->testMaintainerId, ['name' => 'ToRemoveOrg-1']);
+        $project = $this->client->createProject($organization['id'], ['name' => 'ToRemoveProj-1']);
+        $email = 'remove' . uniqid() . '@keboola.com';
+        //PROJECT, ORGANIZATION & MAINTAINER
+        $this->client->addUserToProject($project['id'], ['email' => $email]);
+        $this->client->addUserToOrganization($organization['id'], ['email' => $email]);
+        $this->client->addUserToMaintainer($this->testMaintainerId, ['email' => $email]);
+        //INVITATION
+        $inviteProject = $this->client->createProject($organization['id'], ['name' => 'ToRemoveProj-2']);
+        $this->client->inviteUserToProject($inviteProject['id'], ['email' => $email]);
+
+        // Ensure superadmin is not in org
+        $this->client->removeUserFromOrganization($organization['id'], $this->superAdmin['id']);
+
+        try {
+            $this->client->removeUser($email);
+            $this->fail('An exception should have been thrown!');
+        } catch (Throwable $e) {
+            $expectedMessage = 'Cannot remove "%s" from "%s". Organization must have at least 1 member';
+            $this->assertSame(
+                sprintf(
+                    $expectedMessage,
+                    $email,
+                    $organization['id']
+                ),
+                $e->getMessage()
+            );
+        } finally {
+            $this->client->addUserToOrganization($organization['id'], ['email' => $this->superAdmin['email']]);
```

Comment:
Tohle pridani uzivatele zpet do organizace je zbytecny ne? Se zaklada na zacatku testu, tak ten user?

Reply:
To byl pozůstatek po tom, co jsem ho odebíral i z maintainera.